<launch>
  <include file="$(find realsense_ros_camera)/launch/camera.launch">
    <arg name="fisheye_width" value="640"/>
    <arg name="fisheye_height" value="480"/>
    <arg name="enable_fisheye" value="true"/>

    <arg name="depth_width" value="320"/>
    <arg name="depth_height" value="240"/>
    <arg name="enable_depth" value="true"/>

    <arg name="color_width" value="1920"/>
    <arg name="color_height" value="1080"/> 
    <arg name="enable_color" value="true"/>
  </include>

  <node pkg="rqt_image_view" type="rqt_image_view" name="color_viewer" args="/camera/color/image_raw"/>
  <node pkg="rqt_image_view" type="rqt_image_view" name="depth_viewer" args="/camera/depth/image_raw"/>
  <node pkg="rqt_image_view" type="rqt_image_view" name="fisheye_viewer" args="/camera/fisheye/image_raw"/>
</launch>
